### PR TITLE
admin: reimplement runtime and stats handlers without rapidjson

### PR DESCRIPTION
**Pull request by dio**
This patch reimplements runtime and stats (format=json) handler without rapidjson.

Risk Level: Low
Testing: unit test
Docs Changes: N/A
Release Notes: N/A

Signed-off-by: Dhi Aurrahman <dio@tetrate.io>

### CODE REVIEW

File: source/common/protobuf/utility.cc [hunk 1]
```diff
@@ -610,10 +610,12 @@ ProtobufWkt::Value ValueUtil::structValue(const ProtobufWkt::Struct& obj) {
 }
 
 ProtobufWkt::Value ValueUtil::listValue(const std::vector<ProtobufWkt::Value>& values) {
-  ProtobufWkt::Value val;
+  auto* list = new ProtobufWkt::ListValue();
```
Comment (jmarantz) [inline source/common/protobuf/utility.cc]: Use std::make_unique here and release it when passing it into set_allocated_list_value. That way if there's an early exit or an exception you won't leak.

Comment (dio) [reply source/common/protobuf/utility.cc]: Done. 3ffbd56. Thanks!

**Comment by jmarantz** [comment]
@envoyproxy/senior-maintainers

File: test/server/http/admin_test.cc [hunk 2]
```diff
@@ -56,6 +56,13 @@ using testing::StartsWith;
 namespace Envoy {
 namespace Server {
 
+namespace {
+void expectEqualSerialized(const std::string& str1, const std::string& str2) {
+  EXPECT_TRUE(
+      TestUtility::protoEqual(TestUtility::jsonToStruct(str1), TestUtility::jsonToStruct(str2)));
+}
```
Comment (mattklein123) [inline test/server/http/admin_test.cc]: nit: Can you move this into TestUtility with a better comment on what it does? This is duplicated in two places so far.

**Comment by moderation** [comment]
I think with the completion of this PR the only thing still using it is the Zipkin tracer - https://github.com/envoyproxy/envoy/search?q=rapidjson&unscoped_q=rapidjson

**Comment by dio** [comment]
Yes. I have a PR in flight for that. Will clean it up soon.

**Pull request merged**